packages/framework/src/Model/Order/Processing/OrderProcessorMiddleware/ApplyNominalPromoCodeMiddleware.php

```diff
@@ -93,14 +95,14 @@ public function createAndAddOrderItemData(
      * @param \Shopsys\FrameworkBundle\Model\Order\PromoCode\PromoCode $promoCode
      * @param \Shopsys\FrameworkBundle\Model\Order\PromoCode\PromoCodeLimit\PromoCodeLimit $promoCodeLimit
      * @param \Shopsys\FrameworkBundle\Component\Domain\Config\DomainConfig $domainConfig
-     * @param \Shopsys\FrameworkBundle\Component\Money\Money $totalApplicableProductsPriceAmountWithVat
+     * @param \Shopsys\FrameworkBundle\Model\Pricing\PriceInterface $totalApplicableProductsPrice
      * @return \Shopsys\FrameworkBundle\Model\Order\Item\OrderItemData|null
      */
     protected function createDiscountOrderItemData(
         PromoCode $promoCode,
         PromoCodeLimit $promoCodeLimit,
         DomainConfig $domainConfig,
-        Money $totalApplicableProductsPriceAmountWithVat,
+        PriceInterface $totalApplicableProductsPrice,
```

Comment:
this change should be mentioned in the upgrade notes IMHO

Reply:
I thought we no longer mention changes in method interfaces as those are reported by coding standards.